### PR TITLE
Update microsoft-365-client-support-certificate-based-authentication.md

### PR DESCRIPTION
**Pull request by msbemba**
Made changes to the note as Third-party apps CAN access client certificates from the keychain and have access to the public key in the keychain that points to an external smart card. Reference https://developer.apple.com/documentation/cryptotokenkit/using_cryptographic_assets_stored_on_a_smart_card

### CODE REVIEW

**Comment by msbemba** [comment]
@JohanFreelancer9 Please copy review

**Comment by msbemba** [comment]
Issue#https://github.com/MicrosoftDocs/microsoft-365-docs/issues/7022

File: microsoft-365/enterprise/microsoft-365-client-support-certificate-based-authentication.md [hunk 1]
```diff
@@ -42,7 +42,7 @@ The latest versions of the following clients and platforms support certificate-b
...
+> Edge for iOS and Android supports certificate-based authentication during account add flows. Edge for Android does not support certificate-based authentication when performing authentication against web sites, which are typically intranet sites. <br><br>  In this scenario, a user navigates to a web site (usually on the intranet) where the web site requires the user to authenticate via a certificate. This does not involve modern authentication at all and does not leverage a Microsoft authentication library.
```
Comment (JohanFreelancer9) [inline microsoft-365/enterprise/microsoft-365-client-support-certificate-based-authentication.md]:
```suggestion
> Edge for iOS and Android supports certificate-based authentication during account add flows. Edge for Android does not support certificate-based authentication when performing authentication against websites, which are typically intranet sites. <br><br>  In this scenario, a user navigates to a website (usually on the intranet) where the website requires the user to authenticate via a certificate. This does not involve modern authentication at all and does not leverage a Microsoft authentication library. 
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review.

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by yogkumgit** [comment]
@robmazz Please merge this Pr. Thanks

**Comment by robmazz** [comment]
@yogkumgit and @msbemba - submitted suggested note change to PM team for review.

**Comment by yogkumgit** [comment]
@robmazz Any update?

**Comment by robmazz** [comment]
@yogkumgit - the PM Team is discussing if we want to deprecate all the client service app topics (including this one), should have a decision this week.

**Comment by yogkumgit** [comment]
@robmazz Awaiting your kind response. Thanks

**Comment by robmazz** [comment]
@yogkumgit - this topic has been deprecated

**Pull request closed without merging**